helm-charts/secrets-operator/templates/deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- $ctrlManager := .Values.controllerManager }}
```

Comment:
All changes for the `secrets-operator` helm chart gets auto generated by Kubebuilder. This means that these changes will be overwritten on next release. So feel free to remove or leave it as is